packages/perseus/src/util.ts

```diff
@@ -761,7 +688,6 @@ const Util = {
     textarea,
     unescapeMathMode,
     random,
-    deepClone,
```

Comment:
These three were moved to `perseus-core` since we needed them within the scorer.